src/index.ts

```diff
@@ -89,28 +27,21 @@ export = class OutputDcatUs11 {
     res.set('Content-Type', 'application/json');
 
     try {
-      const hostname = req.hostname;
-      const siteModel = await this.fetchSite(hostname, this.getRequestOptions(portalUrl));
-
-      // Use dcatConfig query param if provided, else default to site's config
-      let dcatConfig = typeof req.query.dcatConfig === 'string'
-        ? this.parseProvidedDcatConfig(req.query.dcatConfig as string)
-        : req.query.dcatConfig;
-
-      if (!dcatConfig) {
-        dcatConfig = _.get(siteModel, 'data.feeds.dcatUS11');
+      const { res: { locals: { feedTemplate, feedTemplateTransforms, nonEditableTemplateFields } } }: {
```

Comment:
You should probably allow for the feedTemplate to arrive on the req.query as well.  req.query.feedTemplate used by default, if not, check in locals.

Reply:
Where do feedTemplateTransforms and nonEditableTemplateFields come from?

Reply:
@rgwozdz `feedTemplateTransforms` and `nonEditableTemplateFields` comes from Koop instance and is part of feed templates.

Reply:
can you link to where we assign to locals?